### ISSUE TITLE
NextcloudVM not accessible anymore after 14 days. 

### ISSUE THREAD

**Issue by bobdig** [nextcloud/vm#1299]
Novice user here: My NextcloudVM installation is only 14 days old. I did installed fresh with scripts.
Now it is not accessible anymore from any browser. 
I did nothing to it other then installed certbot (for wildcard). Auto-Updates were activated. It was just running. 
SSH, Webmin etc still works.

**Comment by enoch85** [comment]
It's probably due to a DHCP renewal of your ISP.

Maybe [the forum](https://help.nextcloud.com) can help you better?

**Comment by bobdig** [comment]
It is al local.

**Comment by enoch85** [comment]
I don't know with the info you gave. It's been running fine for me the past 4 years. 

Maybe you have done something to it that made it go offline? If you enabled automated updates, maybe it failed and Apache needs to be started?

I'm sorry, but "it doesn't work" don't help me much in debugging.

Closing since this is a support matter, and not a bug.

**Comment by bobdig** [comment]
@enoch85 You were right, there was a problem with apache. 
I had created a wildcard-certificate not with that name I had used to create the cert in your script (*.example instead of cloud.example). Later I had deleted the old cert but probably not restarted the vm to find out my error sooner. Today I tried to recreate a cert for cloud.example which failed. I then deleted every cert and started over. But now there was a file called dhparam.pem missing. 
I restored a backup, copied that file, then deleted all certs, started over and finally copied back that file called dhparam.pem and now it works again. Hope this will still work in the future, because I don't know stuff.
Anyway, thanks for your great product vm.
